pets/mysql/galera/test.sh

```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
+kubectl exec mysql-0 -- mysql -u root -e "create database test;"
```

Comment:
GCE deployment tool did it to one of our clusters as well ... 00 on a Casandra node in production ... Wth